apps/src/templates/progress/BubbleBadge.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import FontAwesome from '../FontAwesome';
+
+/**
+ * Icon that indicates an assessment on ProgressBubbles and ProgressPills
+ */
+
+const badgeTypes = ['assessment', 'keepWorking'];
```

Comment:
NIT: i prefer our `utils.makeEnum` pattern for things like this because then you can avoid using string literals as identifiers when you reference the types. e.g. `type === BadgeType.assessment` instead of `type === 'assessment'`